.changeset/rude-llamas-search.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sl-design-system/tabs': patch
+---
+
+Fix NVDA accessibility issues, fix keyboard navigation.
```

Comment:
The way you wrote this, it looks like 2 different things? Could you be more explicit what kind of NVDA accessibility issues you fixed?